### PR TITLE
dynamic Entities Cleanup steps

### PR DESCRIPTION
**Pull request by anadoba**
Entities Cleanup folder that bases on object-reference's `delete_operation_path` optional field.
Additionally, getting rid of Flow-related hardcodes.

### CODE REVIEW

File: postman-generator/src/main/scala/generator/DependantOperationResolver.scala [hunk 1]
```diff
@@ -76,10 +81,20 @@ object DependantOperationResolver extends Logging {
...
-        resources
-          .filter(_.`type` === objRefAttr.resourceType)
-          .flatMap(_.operations)
-          .find(_.method === objRefAttr.operationMethod)
+        val operations =
+          resources
+            .filter(_.`type` === objRefAttr.resourceType)
+            .flatMap(_.operations)
+
+        val referencedOperationOpt = operations.find(_.method === objRefAttr.operationMethod)
```
Comment (ajanczak) [inline postman-generator/src/main/scala/generator/DependantOperationResolver.scala:89]: maybe don't create val and just move it to L95 🤔

File: postman-generator/src/main/scala/generator/Heuristics.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ object Heuristics {
     pathsWithOperation
       .groupBy{case (param, _) => param}
       .mapValues(_.size)
-      .filter(_._1 != "organization")
+      .filter(_._1 != "organization") // TODO: what to do with this?
```
Comment (ajanczak) [inline postman-generator/src/main/scala/generator/Heuristics.scala]: you can just delete this

File: postman-generator/src/main/scala/generator/PostmanCollectionGenerator.scala [hunk 3]
```diff
@@ -228,6 +217,66 @@ object PostmanCollectionGenerator extends CodeGenerator {
...
+    * Fills all path parameters with postman variable references.
+    * If parameter is not specified, it tries to build it from raw path
+    * for example /sth/:id/sth => new 'id' param is created
+    *
+    * @param resolvedService Service with all imports at hand.
+    * @param serviceSpecificHeaders Service specific headers.
+    * @param examplesProvider Examples generator.
+    * @return Dependent entities cleanup folder for postman.
+    */
+  def prepareDependentEntitiesCleanup(
```
Comment (ajanczak) [inline postman-generator/src/main/scala/generator/PostmanCollectionGenerator.scala]: cannot this operation be merged with `prepareDependentEntitiesSetup` and return a Tuple ?  
I think you are running a lot of the same code `DependantOperationResolver.resolve(resolvedService)`. 
Anyway we can leave how it is right now and refactor later if this possible. Your call ;)

**Comment by ajanczak** [comment]
LGTM 👍

**Review by ajanczak — APPROVED** [review]

File: postman-generator/src/main/scala/generator/Heuristics.scala (hunk 2 above)
Comment (anadoba) [reply postman-generator/src/main/scala/generator/Heuristics.scala]: done

File: postman-generator/src/main/scala/generator/PostmanCollectionGenerator.scala (hunk 3 above)
Comment (anadoba) [reply postman-generator/src/main/scala/generator/PostmanCollectionGenerator.scala]: agreed. I've combined them into one method that returns a tuple

File: postman-generator/src/main/scala/generator/DependantOperationResolver.scala (hunk 1 above)
Comment (anadoba) [reply postman-generator/src/main/scala/generator/DependantOperationResolver.scala:89]: i decided to keep it for sake of readability

**Pull request merged**